src/main/scala/is/hail/expr/ir/AggOp.scala

```diff
@@ -9,11 +9,13 @@ import scala.reflect.ClassTag
 
 case class AggSignature(
   op: AggOp,
-  inputType: Type,
   constructorArgs: Seq[Type],
   initOpArgs: Option[Seq[Type]],
-  seqOpArgs: Seq[Type]
-)
+  seqOpArgs: Seq[Type]) {
+  assert(seqOpArgs.nonEmpty)
```

Comment:
This is unnecessary.  In fact, allowing it would fix a small problem: `hl.agg.count()` seqOp wants to take 0 arguments (but right now it takes an artificial `hl.tint64(0)` which is ignored.)

Reply:
Oh, I didn't even think of that.